lib/dor/was_crawl/cdxj_merge_service.rb

```diff
@@ -80,7 +80,10 @@ def clean
       def sort_env_vars
```

Comment:
I wonder if it's worth renaming this method (and the other one) now that its scope is expanding beyond the sort function. Not blocking, tho

Reply:
How has its scope expanded beyond `sort_env_vars`? They are environment variables that are needed for sorting.

Reply:
I originally read this as "env vars to help control how strings sort," without expanding the code context in the PR, and I now see that is not what's going on here. It's "env vars pre-pended to a unix sort command." So it's all good. Merging.